python/setup.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Minor: this file should not be checked in as generated by cmake from `setup.py.in`. I think I'll change this as this is confusing and have `setup.py` include `version.py.in` which specifies the version.

Reply:
I've removed it. Don't know how it got included, since its in `.gitignore`. Dodgy merge at some point.